applications/applications_interface.py

```diff
@@ -472,7 +462,7 @@ def ImportApplications(kernel, applications_path=application_directory):
         wind_turbine_application = KratosMultiScaleApplication()
         kernel.ImportApplication(MultiScaleApplication)
         print("KratosMultiScaleApplication sucessfully imported")
-		
```

Comment:
this file should be removed overall. again as a part of a different Pr

Reply:
@RiccardoRossi I have the feeling that it is time to clean up and update our python interface, maybe as a goal for the next release, and removing this should be part of it.